php-nginx/entrypoint.sh

```diff
@@ -133,9 +132,7 @@ chmod -R 550 ${DOCUMENT_ROOT}
 # Change the www-data's shell back to /usr/sbin/nologin
 chsh -s /usr/sbin/nologin www-data
 
-# Put a stricter php-cli.ini
-if [ -f "${PHP_DIR}/lib/php-cli-strict.ini" ]; then
-    mv ${PHP_DIR}/lib/php-cli-strict.ini ${PHP_DIR}/lib/php-cli.ini
-fi
+# Enable suhosin
+${PHP56_DIR}/bin/php56-enmod suhosin
```

Comment:
so we enable suhosin when the container starts (i.e. in the entrypoint), which is fine with me, but I don't like how this is run even if the PHP version isn't php56. I guess the reasoning here is we'd have to run `detect_php_version` again, and as this doesn't harm anything we just leave it. Is that accurate?